Src/DependencyCollector/Net45.Tests/DependencyTrackingTelemetryModuleTest.cs

```diff
@@ -64,7 +79,20 @@ public void TestHttpPostRequestsAreCollected()
         public void TestHttpRequestsWithQueryStringAreCollected()
         {
             ITelemetry sentTelemetry = null;
-            var channel = new StubTelemetryChannel { OnSend = telemetry => sentTelemetry = telemetry };
+
+            var channel = new StubTelemetryChannel
+            {
+                OnSend = telemetry =>
+                {
+                    // The correlation id lookup service also makes http call, just make sure we skip that
+                    DependencyTelemetry depTelemetry = telemetry as DependencyTelemetry;
+                    if (depTelemetry != null && !depTelemetry.Data.StartsWith(FakeProfileApiEndpoint, StringComparison.OrdinalIgnoreCase))
```

Comment:
Minor: It looks like Data can be null if Dependency telemetry was initialized with default constructor. This particular test should not be affected.